taichi/transforms/type_check.cpp

```diff
@@ -414,8 +414,8 @@ class TypeCheck : public IRVisitor {
   void visit(GetElementStmt *stmt) override {
     TI_ASSERT(stmt->src->is<FuncCallStmt>());
     auto *func = stmt->src->as<FuncCallStmt>()->func;
-    TI_ASSERT(stmt->index < func->rets.size());
-    stmt->ret_type = func->rets[stmt->index].dt;
+    TI_ASSERT(stmt->index[0] < func->rets.size());
```

Comment:
Same as above, what's special about index 0, leave a comment here or make it a named parameter